klippa/src/variations.rs

```diff
@@ -419,6 +422,102 @@ fn collect_region_refs(
     }
 }
 
+pub(crate) struct DeltaSetIndexMapSerializePlan<'a> {
+    outer_bit_count: u8,
+    inner_bit_count: u8,
+    output_map: &'a FnvHashMap<GlyphId, u32>,
+    map_count: u32,
+}
+
+impl<'a> DeltaSetIndexMapSerializePlan<'a> {
+    pub(crate) fn new(
+        outer_bit_count: u8,
+        inner_bit_count: u8,
+        output_map: &'a FnvHashMap<GlyphId, u32>,
+        map_count: u32,
+    ) -> Self {
+        Self {
+            outer_bit_count,
+            inner_bit_count,
+            output_map,
+            map_count,
+        }
+    }
+
+    pub(crate) fn get_width(&self) -> u8 {
+        (self.outer_bit_count + self.inner_bit_count + 7) / 8
+    }
+
+    pub(crate) fn get_inner_bit_count(&self) -> u8 {
+        self.inner_bit_count
+    }
+
+    pub(crate) fn get_output_map(&self) -> &'a FnvHashMap<GlyphId, u32> {
+        self.output_map
+    }
+
+    pub(crate) fn get_map_count(&self) -> u32 {
+        self.map_count
+    }
```

Comment:
By convention we generally leave the `get_` prefix off of getters (but use `set_` for setters)